django/contrib/auth/tokens.py

```diff
@@ -25,12 +26,20 @@ def _set_secret(self, secret):
 
     secret = property(_get_secret, _set_secret)
 
+    def _get_fallbacks(self):
+        return self._secret_fallbacks or settings.SECRET_KEY_FALLBACKS
```

Comment:
A None check would be required here (and not just look for truthiness of self._secret_fallbacks). 

I.e. using `SECRET_KEY_FALLBACKS = ['a']` and setting `generator.secret_fallbacks = []` would make the getter `generator.secret_fallbacks` return `['a']` and not `[]`.

Reply:
Good catch! Thank you